docs/articles/Getting-Started.md

```diff
@@ -0,0 +1,386 @@
+# Getting Started
+
+One App is a web application that combines together multiple experiences into a single,
+cohesive runtime. These experiences, or Holocron modules as we call them empower teams
+to craft a rich user experience that can be used to compose parts of a web page or as
+route destinations all on their own. One App was built from the ground up with Holocron
+and its micro frontend architecture, allowing engineers to seamlessly update each module
+independently and collaborate across teams working together to create a product. While
+One App comes with a security standard in practice, progressive web app capability and
+many other features, its feature set can be configured by Holocron modules to your
+exact specification.
+
+This guide will get us started with One App and will go over some of the
+fundamental concepts of the framework. We will start from the beginning
+by creating our first Holocron module, then we will go through a few
+One App essentials to get up and running quickly.
+
+## 📖 Table of Contents
+* [Generating a Module](#generating-a-module)
+* [Running One App](#running-one-app)
+* [Adding CSS Styles](#adding-css-styles)
```

Comment:
i really like how you've broken this down into different sections

Reply:
Thank you!